R/reportEmi.R

```diff
@@ -1878,8 +1879,45 @@ if (!is.null(vm_plasticsCarbon)) {
                         - out[, , "Emi|CO2|CDR|BECCS|Industry (Mt CO2/yr)"],
                         "Emi|CO2|Gross|Energy|+|Demand (Mt CO2/yr)"))
 
+###########################################
+## Gross emissions in Energy|Waste sector##
+###########################################
+# fixme: check that CDR addition makes sense
+if (!is.null(vm_plasticsCarbon)){
+# calculate gross emissions in energy waste sector
+  out <- mbind(out,
+               # total gross energy waste emissions
+               setNames(out[, , "Emi|CO2|Energy|+|Waste (Mt CO2/yr)"]
+                        - out[, , "Emi|CO2|CDR|Materials|+|Plastics (Mt CO2/yr)"],
+                        "Emi|CO2|Gross|Energy|+|Waste (Mt CO2/yr)"))
+}
+
+###########################
+## total gross variables ##
+###########################
+# if feedstocks are available
+if (!is.null(vm_plasticsCarbon)){
+ out <- mbind(out,
+               # total gross energy emissions
+               setNames(out[, , "Emi|CO2|+|Energy (Mt CO2/yr)"]
+                        - out[, , "Emi|CO2|CDR|Industry CCS|Synthetic Fuels (Mt CO2/yr)"]
+                        - out[, , "Emi|CO2|CDR|BECCS (Mt CO2/yr)"]
+                        - out[, , "Emi|CO2|CDR|Materials|+|Plastics (Mt CO2/yr)"],
```

Comment:
I think this is a general issue we should discuss in the IAMC emissions group with everybody, that is, accounting of long-term CCU. I tend to be in favor of the convention that carbon stored in materials should be deduced from industry / energy emissions because that is how the IPCC emissions accounting defines energy-related emissions (i.e. only emissions from combustion). But I am happy to discuss further.